routes/accounts.js

```diff
@@ -3,6 +3,8 @@ const express = require( 'express' )
 const encryptPassword = require( '../auth/encryptPassword' )
 const passport = require( '../auth/passport' )
 const RegistrationEmail = require( '../src/mail/registration_email.js' )
+const validateEmail = require( '../src/mail/validate_email.js')
+const registerUser = require( './accounts/register_user')
```

Comment:
Please ensure there is a space between parameter and opening and closing parens (missing space at closing parens here)